streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedTasks.java

```diff
@@ -124,7 +124,7 @@ void initializeNewTasks() {
                 it.remove();
             } catch (final LockException e) {
                 // made this trace as it will spam the logs in the poll loop.
-                log.trace("{} Could not create {} {} due to {}; will retry", logPrefix, taskTypeName, entry.getKey(), e.getMessage());
+                log.debug("{} Could not create {} {} due to {}; will retry in the next run loop", logPrefix, taskTypeName, entry.getKey(), e.getMessage());
```

Comment:
not sure changing this to `debug` is a great idea. It can end up spamming the logs as the loop is tight and it might take some time for the other threads/instances to release the locks. Hence the comment above it

Reply:
Makes sense. Ack.